data/mapping/mapping.json

```diff
@@ -4,31 +4,25 @@
       "analyzer":{
         "gp_name_analyzer":{
```

Comment:
It's the custom name tokenizer which means it has to be a custom analyzer. The use of a custom tokenizer is preferred to the standard tokenizer as it handles searching for names like "O'Reilly" properly ([source](https://twitter.com/ErikRose/status/314135579266777089)).